#!/bin/sh -e
## 30-slurpd.pm.dpatch by Steffen Joeris <steffen.joeris@skolelinux.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Here we correct the path for kolabconf

if [ $# -lt 1 ]; then
echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
      -patch) patch $patch_opts -p0 < $0;;
      -unpatch) patch $patch_opts -p0 -R < $0;;
      *)
      echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
      exit 1;;
esac

exit 0

@DPATCH@
--- slurpd.pm.orig	2006-02-09 16:15:22.405392842 +0100
+++ Kolab-LDAP-Backend-slurpd/slurpd.pm	2006-02-09 16:15:38.309247252 +0100
@@ -277,7 +277,7 @@ sub run
                 $changes = 0;
                 Kolab::log('SD', 'Change detected w/ no pending LDAP messages; reloading services if needed');
                 Kolab::LDAP::sync;
-                system($Kolab::config{'prefix'} . '/sbin/kolabconf');
+                system('/sbin/kolabconf');
             }
 
             Kolab::log('SD', 'Waiting for LDAP updates');
